### ISSUE TITLE
NPE on switch off word-wrap

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#1931]
I'm getting an intermittent NPE

````
java.lang.NullPointerException
	at org.eclipse.nebula.jface.gridviewer.GridTableViewer.setSelectionToWidget(GridTableViewer.java:485)
	at org.eclipse.jface.viewers.StructuredViewer.setSelection(StructuredViewer.java:1723)
	at org.eclipse.jface.viewers.Viewer.setSelection(Viewer.java:394)
	at org.mwc.cmap.NarrativeViewer.NarrativeViewer.calculateGridRawHeight(NarrativeViewer.java:199)
	at org.mwc.cmap.NarrativeViewer.NarrativeViewer.setWrappingEntries(NarrativeViewer.java:217)
	at org.mwc.cmap.NarrativeViewer.app.NViewerView$8.run(NViewerView.java:430)
	at org.eclipse.jface.action.Action.runWithEvent(Action.java:498)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:584)
	at org.eclipse.jface.action.ActionContributionItem.access$2(ActionContributionItem.java:501)
	at org.eclipse.jface.action.ActionContributionItem$6.handleEvent(ActionContributionItem.java:452)
````

If I start without the Narrative Viewer open, and open the the Viewer, the word-wrap on/off works fine.

But, if I start with the Viewer already open (from the previous session) changing word-wrap throws the above NPE.

**Comment by theanuradha** [comment]
@IanMayo  I added fix as part of #1930 can you please check it should be fine now

**Comment by IanMayo** [comment]
My mistake - yes, it's already fixed in master.
